wemake_python_styleguide/violations/naming.py

```diff
@@ -439,3 +441,33 @@ class SameAliasImportViolation(ASTViolation):
     #: Error message shown to the user.
     error_template = 'Found same alias import "{0}"'
     code = 113
+
+
+class UnderScoredNumberNameViolation(SimpleViolation):
+    """
+    Forbids to have names with underscored numbers pattern.
+
+    Reasoning:
+        This is done for consistency in naming.
+
+    Solution:
+        Rename your variable or modules to not include underscored numbers
+
+    This rule checks: variables, and modules.
+
+    Example::
+
+        # Correct:
+        star_wars_episode2
```

Comment:
Can you please also list that `iso_123_456` is a valid name?

Reply:
Added.